desktop/release-notes.md

```diff
@@ -24,12 +24,68 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.13.1
+2022--
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
```

Comment:
All the download links will need to be changed in this page, plus the checksums (which you already know about :) )